jQuery/src/index.js

```diff
@@ -0,0 +1,101 @@
+import { defaultData } from './data.js';
+
+$(() => {
+    $('#scheduler').dxScheduler({
+        dataSource: defaultData,
+        views: ['week'],
+        currentView: 'week',
+        currentDate: new Date(2020, 10, 25),
+        firstDayOfWeek: 0,
+        startDayHour: 9,
+        height: '100%',
+        width: '100%',
+        onAppointmentAdding: (e) => {
+            handleAppointmentActions(e, e.appointmentData, popup);
+        },
+        onAppointmentUpdating: (e) => {
+            handleAppointmentActions(e, e.newData, popup);
+        }
+    })
+});
+
+const popupContentTemplate = function () {
+    return $('<div>').append(
+        $(`<p>There is a recurrent appointment in this cell.</p>`),
+    );
+};
+
+const popup = $('#popup').dxPopup({
+    contentTemplate: popupContentTemplate,
```

Comment:
Let's move ```popupContentTemplate``` function to ```dxPopup``` config:

```suggestion
const popup = $('#popup').dxPopup({
       contentTemplate: () => $('<div>').append(
            $(`<p>There is a recurrent appointment in this cell.</p>`),
        ),
```